fixtures/ios_app_with_external_objc_packages/Project.swift

```diff
@@ -0,0 +1,17 @@
+import ProjectDescription
+
+let project = Project(
+    name: "App",
+    targets: [
+        Target(
+            name: "App",
+            platform: .iOS,
+            product: .app,
+            bundleId: "io.tuist.App",
+            sources: ["Sources/**"],
+            dependencies: [
+                .external(name: "GoogleSignIn")
+            ]
+        )
+    ]
```

Comment:
I think we already have way too many fixtures, and we should aim to remove/merge some of them.
Can't this be just part of `app_with_spm_dependencies`?